src/components/status-badges/component-secret-status-badge.tsx

```diff
@@ -9,24 +9,34 @@ import {
 
 import { SecretStatus } from '../../models/secret-status';
 
+export type ComponentSecretStatusBadgeProps = {
+  status: SecretStatus;
+};
```

Comment:
remember to change the PropTypes.ValidationMap typing as well
`PropTypes.ValidationMap<ComponentSecretStatusBadgeProps>`

```suggestion
export interface ComponentSecretStatusBadgeProps {
  status: SecretStatus;
}
```